target/linux/ramips/image/mt76x8.mk

```diff
@@ -809,10 +809,15 @@ TARGET_DEVICES += tplink_re220-v2
 
 define Device/tplink_re305-v1
   $(Device/tplink-safeloader)
-  IMAGE_SIZE := 6016k
+  IMAGE_SIZE := 7680k
+  KERNEL_SIZE := 6016k
   DEVICE_MODEL := RE305
   DEVICE_VARIANT := v1
   DEVICE_PACKAGES := kmod-mt76x2
+  DEVICE_COMPAT_VERSION := 1.1
+  DEVICE_COMPAT_MESSAGE := Partition design has changed compared to older versions due to size restrictions and unsused flash. \
+	Upgrade via sysupgrade mechanism is not possible, so new installation via TFTP is required.
```

Comment:
Please add `IMAGES := sysupgrade.bin` to disable the `factory` image.

TFTP flash write installation will damage the `config` partition if image is too big. We need to boot initramfs and then upgrade sysupgrade image.

Reply:
@DragonBluep thanks for the hint. I updated my PR and checked the outcome of the build